accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -324,6 +334,7 @@ public static void main(String[] args)
     @Test
     public void testOne()
     {
+        for (; true; )
```

Comment:
Is this supposed to be here still?

Reply:
nope... I add it to run for a long time... forgot to remove